students/daehyun/westargram/users/views.py

```diff
@@ -54,10 +55,15 @@ def post(self, request):
             if not re.match(PASSWORD_RULE, password):
                 return JsonResponse ({'message' : 'INVALID_USER_PASSWORD'}, status = 400)
             
-            if not User.objects.filter(email = data['email'], password = data['password']).exists():
+            member = User.objects.get(email = email)
```

Comment:
변수명 올바르게 작성해주세요. 만약 `member`로 작성하실거였다면 테이블명도 `members`라던지 관련된 이름이어야 명확할 것 같습니다
```suggestion
            user = User.objects.get(email = email)
```